### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by jburel**
This PR removes dependencies not needed that were creating conflicts with the upgrade of testNG

### CODE REVIEW

File: src/test/resources/test.xml [hunk 1]
```diff
@@ -12,9 +12,5 @@
   <dependencies defaultconfmapping="test->*">
     <dependency name="server" rev="${omero.version}" changing="true"/>
     <dependency name="common-test" rev="${omero.version}" changing="true"/>
-    <dependency name="blitz-test" rev="${omero.version}" changing="true"/>
```
Comment (sbesson) [inline src/test/resources/test.xml:15]: :+1: It's unclear to me how this ever worked as this is a dependency inversion

**Review by sbesson — COMMENTED** [review]
The tests are green with this PR included and it makes sense to drop the last remaining mentions of `log4j` bindings (even if these are in the tests).

As this fully drops the `org.apache.directory`, I wonder what the impact might be. Are there some relevant tests that are currently excluded? It looks like these dependencies were moved to the test components in https://github.com/ome/openmicroscopy/pull/1006/commits/56a4686a678cc4684505908ad056ad6ce07c3bed but I could not find relevant code consuming these.

File: src/test/resources/test.xml (hunk 1 above)
Comment (jburel) [reply src/test/resources/test.xml:15]: It is unclear to me too.

**Comment by jburel** [comment]
> As this fully drops the `org.apache.directory`, I wonder what the impact might be. Are there some relevant tests that are currently excluded? It looks like these dependencies were moved to the test components in [ome/openmicroscopy@56a4686](https://github.com/ome/openmicroscopy/commit/56a4686a678cc4684505908ad056ad6ce07c3bed) but I could not find relevant code consuming these.

I could not find any relevant tests using the ``apacheds`` dependency

**Review by sbesson — APPROVED** [review]
Looks good to me overall. Maybe @joshmoore can remember the history of the `apacheds` dependency and where it might be used

File: src/test/resources/test.xml (hunk 1 above)
Comment (joshmoore) [reply src/test/resources/test.xml:15]: I would have to dig up the specifics, but the test dependency graph definitely differs from the jar dependency graph, or at least did in the mono repo.

File: build.gradle [hunk 2]
```diff
@@ -24,18 +24,9 @@ ext {
 }
 
 dependencies {
-    testImplementation("org.testng:testng:7.5") {
-        //exclude to pass compilation step due to conflicting versions in the path
-        exclude group: "org.slf4j", module: "slf4j-log4j12"
-    }
+    testImplementation("org.testng:testng:7.5")
     testImplementation("jmock:jmock:1.+")
-    testImplementation("org.apache.directory.server:apacheds-all:1.5.7")
```
Comment (joshmoore) [inline build.gradle:32]: Surprising that this can be removed with no impact. Did you happen to notice if the related code was removed/disabled?

**Comment by joshmoore** [comment]
There certainly _were_ tests at some point. I don't remember removing/disabling them, so I suspect this may have happened in the migration 😨

**Comment by jburel** [comment]
--exclude

**Comment by jburel** [comment]
Failure in new Jenkins is not related to this PR.

File: src/test/resources/test.xml (hunk 1 above)
Comment (jburel) [reply src/test/resources/test.xml:15]: logically it also does not make sense to have the dependency on blitz-test since omero-server is built before.

**Pull request merged**